ts/features/bonus/cgn/components/detail/eyca/EycaInformationComponent.tsx

```diff
@@ -8,8 +8,7 @@ import ButtonDefaultOpacity from "../../../../../../components/ButtonDefaultOpac
 import { Label } from "../../../../../../components/core/typography/Label";
 import { showToast } from "../../../../../../utils/showToast";
 import { VSpacer } from "../../../../../../components/core/spacer/Spacer";
-
-const EYCA_DISCOUNTS_URL = "https://www.eyca.org/discounts/it";
+import { EYCA_WEBSITE_URL } from "../../../utils/constants";
```

Comment:
```suggestion
import { EYCA_WEBSITE_DISCOUNTS_PAGE_URL } from "../../../utils/constants";
```
What you think about this? Isn't it a specific page/section of the website?

Reply:
You can also have the base website url and the discount page one relative to the base one.

Reply:
Previously in that page the link was to EYCA discounts page but by design is changed redirecting to the EYCA **homepage** website. 

I agree with you to build the discount page from website url as base path and I've done that with this commit: [122d472](https://github.com/pagopa/io-app/pull/4689/commits/122d472cd96d6010c134351178fba3259c60a7d2)